docs/jsonapi-spec.textile

```diff
@@ -438,6 +571,15 @@ h4(#commandUpdateManyOptions). updateMany Command Options
 |_. Option |_. Type |_. Description |
 | @upsert@ | Boolean | When @true@ if no documents match the @filter@ clause the command will create a new _empty_ document and apply the @update@ clause to the empty document. If the @_id@ field is included in the @filter@ the new document will use this @_id@, otherwise a random value will be used see "Upsert Considerations":#considerationsUpsert for details. When false the command will only update a document if one matches the filter. Defaults to @false@.|
 
+h4(#commandUpdateManyResponse). updateMany Command Response
+
+|_. Response Element |_. Description |
+| @data@ | Not present. |
+| @status@ |Present with fields: @matchedCount@ count of documents that matched the @filter@, @modifiedCount@ count of document updated (may be less than match count), @upsertedId@ if a document was upserted (when @matchCount@ is zero) not present otherwise.|
```

Comment:
As I understood idea is to insert a single doc with upsert in case match count is zero? Seems fine :+1: